Gemfile

```diff
@@ -14,6 +14,8 @@ gem 'devise'
 gem "haml-rails", "~> 2.0"
 gem 'simple_form'
 gem 'twitter-bootstrap-rails', :git => 'git://github.com/seyhunak/twitter-bootstrap-rails.git'
```

Comment:
Those gem was last updated 1.5 years ago
![image](https://user-images.githubusercontent.com/54136347/124726478-77f8ca00-df16-11eb-8801-88c9a4c54c0e.png)
I guess better use another gem for bootstrap
https://github.com/twbs/bootstrap-rubygem